lib/entry/LiveEntry.js

```diff
@@ -130,6 +131,30 @@ function saveFlavorDownloadersConfiguration() {
     });
 }
 
+function getEntryServerNodeId() {
+    var getLiveEntryServerNodeIdPromise =
```

Comment:
yes. As I moved the function from StateManager to LiveEntry as I think it is the correct place. Removed it from stateManager.